javascript/classes.json

```diff
@@ -0,0 +1,244 @@
+{
+  "javascript": {
+    "classes": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Classes",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": [
+            {
+              "version_added": "49"
+            },
+            {
+              "version_added": "42",
+              "notes": "Requires strict mode. Non-strict mode support is behind the flag \"Enable Experimental JavaScript\", disabled by default."
+            }
+          ],
+          "chrome_android": {
+            "version_added": "56"
+          },
+          "edge": {
+            "version_added": "13"
+          },
+          "edge_mobile": {
+            "version_added": "13"
+          },
+          "firefox": {
+            "version_added": "45"
+          },
+          "firefox_android": {
+            "version_added": "45"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "ie_mobile": {
+            "version_added": false
+          },
+          "nodejs": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": "43"
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": "9"
+          },
+          "safari_ios": {
+            "version_added": "9"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "constructor": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Classes/constructor",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": [
+              {
+                "version_added": "49"
+              },
+              {
+                "version_added": "42",
+                "notes": "Requires strict mode. Non-strict mode support is behind the flag \"Enable Experimental JavaScript\", disabled by default."
```

Comment:
I think we should either, merge both entries:

"chrome": { 
  "version_added": "49",
  "notes": "From Chrome 42 to 48 was only working in strict mode"
}

or to make a subfeature:
"javascript"."classes"  (with "42"/no comment)
"javascript"."classes"."non_strict" (with 49, and the pref entry with 42)

(and same values of Fx/IE/Edge/Safari)

Reply:
I'm merging the two entries.